shopyo/modules/admin/models.py

```diff
@@ -7,17 +7,22 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 from shopyoapi.init import db
 from flask_login import UserMixin
+from uuid import uuid4
 
 
 class Users(UserMixin, db.Model):
     """ Users model """
 
     __tablename__ = "users"
     id = db.Column(db.String(10), primary_key=True)
-    name = db.Column(db.String(100))
+    username = db.Column(db.String(100), unique=True)
     password = db.Column(db.String(128))
     admin_user = db.Column(db.Boolean, default=False)
 
+    def __init__(self, **kwargs):
+        super(Users, self).__init__(**kwargs)
+        self.id = str(uuid4())
+
     def set_hash(self, password):
         self.password = generate_password_hash(password, method="sha256")
 
```

Comment:
While you are at it, could you add the insert, update and delete method? Also can you change the user management module aka the admin module and add the change from id to username?

Reply:
@Abdur-rahmaanJ sure thing will get right on that.

Reply:
@Abdur-rahmaanJ So I have added a save and delete method to the user model. And made changes to the admin view.